client/json-rpc/Cargo.toml

```diff
@@ -23,4 +23,4 @@ libra-workspace-hack = { path = "../../common/workspace-hack", version = "0.1.0"
 
 [features]
 default = ["tls"]
-tls = ["reqwest/rustls-tls"]
+tls = ["reqwest/default-tls"]
```

Comment:
This isn't needed. default-tls is in the default feature.

Reply:
`reqwest` has `default_features=false` above however?

Reply:
Sorry, I misread the diff and thought this was in workspace-hack dir.